### DISCUSSION TITLE
2 unterschiedliche Geräte melden in Minutenabstand "Multicast (re)joined"

### DISCUSSION THREAD

**Discussion by Joylancer** [arendst/Tasmota#21251] [Q&A]
### PROBLEM DESCRIPTION
Ich habe ein Problem mit dem Wifi Reconnect bei der letzten Dev .
2 Geräte, einmal
Sonoff Dual R3 V2 (ESP32-D0WD-V3 v3.0/tasmota32.bin)
und beim
Shelly Plus 2PM v0.1.9 (ESP32-U4WDH-S v3.0/tasmota32solo1.bin).

Bemerkt habe ich es im Sonoff Adapter, das beide sich permanent reconnected haben.
Bin jetzt zurück auf die Release-Version (13.4.0), damit keine Probleme.
Fehler bei beiden: Multicast (re)joined
Vielleicht könnt ihr damit etwas anfangen.
Alle anderen Geräte (ESP8266) laufen ohne Probleme.
Falls ihr weitere Logs braucht, kann ich gerne zur Verfügung stellen, komme allerdings erst nach 20 Uhr dazu.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ x] Device used (e.g., Sonoff Basic): Sonoff Dual R3 V2 // Shelly Plus2PM V0.1.9
- [ x] Tasmota binary firmware version number used: 13.4.0.4
  - [x ] Pre-compiled
  - [ ] Self-compiled
- [x ] Flashing tools used: __ota__
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
This is an English only repo. Moved to discussions. Issue is not reproducable.
